README.md

```diff
@@ -1,222 +1,60 @@
-
-## Handling UI
-
-### Creating UI pages for models
-
-To create UI pages for a model, type:
-
-```bash
-nrp ui model create --model <model-name> <ui-name>
-```
-
-The `ui-name` is optional, if not specified, it will be the same
-as the `model-name`. The command will ask a couple of questions
-and will create jinjax templates and react pages for displaying 
-a listing of the model, a detail page and a form for creating
-and editing the model.
-
-### Creating UI pages for custom endpoints
-
-To create UI pages for a custom endpoint, type:
-
-```bash
-nrp ui page create <page-name> <page-endpoint>
-```
-
-The `page-endpoint` is the endpoint of the page, for example
-`/about` or `/search`. The `page-name` is the name of the page,
-for example `about` or `search`.
-
-If `page-endpoint` is not specified, it will be the same as
-`page-name`.
-
-The command will create a jinjax template for the page and register
-the page to the flask application.
-
-If you run the command with `--react` option, it will also create
-react endpoint for the page and reference it from the jinjax template.
-
-===
-
-Here's Mirek from the same company (CESNET) as mirekys. I'd just add that for our
-ongoing "Czech National Repository Platform" project the direct multipart
-upload to S3 is vital, as our users have already been using this form of
-upload both in our repositories based on older versions of Invenio 3 and in
-the LTP service we provide.
-We have started migrating our codebase to RDM 12 and the plan was to have 
-the S3 layer ported by the end of Q1. With what you do we'd rather not split 
-forces and use your solution. If you'd like a hand with development or testing, 
-let us know.
+## Overview
+
+Document part of the Czech National Repository (NR) is a digital repository system designed to manage and preserve digital content. It provides a platform for storing, sharing, and accessing
+digital documents, ensuring their long-term availability and integrity.
+
+## Installation and Setup
+
+To install and set up this repository, follow these steps:
+
+1. Create a `scripts/release-<version>.sh` file inside the `scripts` directory.
+2. Build the docker image with the repository. 
+   - create a deployment branch called `deployment/<version>`
+   - go to https://github.com/oarepo/oarepo-deployment and run an action there to
+     build the image
+3. Ask devops to deploy the image to the production environment via the support channel.
+   Pass them the following information:
+    - the name of the repository (e.g., `nr-docs`)
+    - version of the repository (e.g., `1.0.0`)
+4. Devops will deploy the repository on a staging environment.
+    - The repository will be available at https://docs.staging.narodni-repozitar.cz
+5. The devops will run the `scripts/release-<version>.sh`. The initial deployment version
+   will create the following:
+   - users
+   - roles
+   - communities
+   - import initial data
+   Subsequent script might, for example, convert the data
```

Comment:
<img width="604" height="71" alt="Screenshot 2025-07-31 at 21 31 16" src="https://github.com/user-attachments/assets/5fcee864-99a9-471c-b8bd-b875cab3d238" />

add \n to fix formatting